packages/lib-core/src/runtime/PluginLoader.ts

```diff
@@ -107,10 +107,11 @@ export type PluginLoaderOptions = Partial<{
   fixedPluginDependencyResolutions: Record<string, string>;
 
   /**
-   * Shared scope object for initializing `PluginEntryModule` containers.
+   * webpack share scope object for initializing `PluginEntryModule` containers.
```

Comment:
@vojtechszocs nit - can we capitalize the `W` in webpack?

Reply:
Synced with Vojtech offline. Webpack has taken the stance on using lowercase `w` when referring to webpack.